<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CcpaSpancExportFactures</class>
 <widget class="QDialog" name="CcpaSpancExportFactures">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>695</width>
    <height>597</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Facturation</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>681</width>
     <height>577</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="1">
       <widget class="QLineEdit" name="civilite">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="nom">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Civilité / Nom / Prénom</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="adresse1">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Adresse2</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>CP / Ville / Pays</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLineEdit" name="prenom">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Adresse 1</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="adresse2">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="cd">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLineEdit" name="ville">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Numéro de dossier</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="noDossier"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Contrôle</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="CcpaSpancCombo" name="controle"/>
      </item>
      <item row="5" column="2">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Tarif</string>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="CcpaSpancCombo" name="tarif"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTableView" name="listView"/>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>Supprimer les lignes sélectionnées</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="nbDossiersL">
        <property name="text">
         <string>Nombre de dossiers</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="nbDossiers"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="totalL">
        <property name="text">
         <string>Total facturé:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="totalFacture"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="numRoDeBordereauLabel">
        <property name="text">
         <string>Numéro de bordereau</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="noBordereau"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="libellDeLaFacturationLabel">
        <property name="text">
         <string>Libellé de la facturation</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="labelFactu"/>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="noTitre"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="numRoDuTitreLabel">
        <property name="text">
         <string>Numéro du titre</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QDateEdit" name="dateFactu"/>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="dateDeFacturationLabel">
        <property name="text">
         <string>Date de facturation</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_3">
      <property name="text">
       <string>Procéder à la facturation</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout_2"/>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CcpaSpancCombo</class>
   <extends>QComboBox</extends>
   <header>ccpaspanccombo.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>noDossier</tabstop>
  <tabstop>civilite</tabstop>
  <tabstop>nom</tabstop>
  <tabstop>prenom</tabstop>
  <tabstop>adresse1</tabstop>
  <tabstop>adresse2</tabstop>
  <tabstop>cd</tabstop>
  <tabstop>ville</tabstop>
  <tabstop>controle</tabstop>
  <tabstop>tarif</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>listView</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>nbDossiers</tabstop>
  <tabstop>totalFacture</tabstop>
  <tabstop>noBordereau</tabstop>
  <tabstop>noTitre</tabstop>
  <tabstop>labelFactu</tabstop>
  <tabstop>pushButton_3</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CcpaSpancExportFactures</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CcpaSpancExportFactures</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
